README.md

```diff
@@ -98,6 +98,98 @@ function Component(){
 }
 ```
 
+## Using the Address component
```

Comment:
I probably wouldn't put this in the README, documentation for components should live somewhere else. Something like storybook would be good for this or perhaps some component tests with enzyme and jest. I guess for now the description could go in the PR description? Thoughts?

Reply:
I'd love to setup storybook if need be. But for now, will stick to the README

Reply:
Yh i'd create a separate README and put it in same folder as the component though

Reply:
The right solution here is to move the component to its own repository with its own README.

Reply:
@Arachnid please take a look https://github.com/iamonuwa/ENS-Address